components/ui/web/components/content/compute/sessionManagement/newSession.tsx

```diff
@@ -1,23 +1,14 @@
 import { FC, useMemo, useState } from 'react';
 import { useRouter } from 'next/router';
-import styled from 'styled-components';
-import {
-  Button,
-  CircularProgress,
-  IconButton,
-  TextField,
-  Typography
-} from '@mui/material';
-import { ArrowBackIos as ArrowBackIcon } from '@mui/icons-material';
 import { useQuery, ApolloError } from '@apollo/client';
+import { Chip, IconButton } from '@mui/material';
+import { Close as CloseIcon } from '@mui/icons-material';
+import styled from 'styled-components';
 
-import { DomainAccordionSummary } from 'components/content/compute/newSession/domainAccordion';
-import { ImageAccordionSummary } from 'components/content/compute/newSession/imageAccordion';
-import { DataVolAccordionSummary } from 'components/content/compute/newSession/dataVolumeAccordion';
-import { UserVolAccordionSummary } from 'components/content/compute/newSession/userVolumeAccordion';
 import { DataVolume, Domain, Image, UserVolume } from 'src/graphql/typings';
 import { GET_DOMAINS } from 'src/graphql/domains';
-import { LoadingAnimation } from 'components/common/loadingAnimation';
+
+import { NewResource, NewSessionType } from 'components/content/newResource/newResource';
```

Comment:
Mentioning here since it is the first place I see, but I think the name `NewResource` is probably too generic. Looks like this is a specification particular to compute. Could it rather be `NewSession` or `NewSessionSpec` or something - this would match some of the other naming better (session name, session type)